src/components/TableCell/TableCell.tsx

```diff
@@ -81,7 +80,6 @@ export const TableCell = ({
   const componentClassName = clsx(
     styles['table__cell'],
     align === 'left' && styles['table__cell--align-left'],
-    align === 'insetLeft' && styles['table__cell--align-inset-left'],
```

Comment:
Should `insetLeft` be removed from the type definition on line 10?

Reply:
yes, removed